lib/std/os/linux/test.zig

```diff
@@ -84,6 +84,8 @@ test "statx" {
         else => unreachable,
     }
 
+    if (builtin.cpu.arch == .riscv32) return; // No fstatat, so the rest of the test is meaningless.
```

Comment:
I thought about that, but we are still testing the `statx` syscall, even if we don't run the rest of the test. I don't know. Bit of an odd case.

Reply:
Even though we're testing `statx`, it should be a `return error.SkipZigTest`. We don't count it as a pass since the entire test didn't pass, but there is the logic that will trigger a failure above which we can test. Note that a `return` will count the test as a pass.

At least this is the current status quo in the testing suite.